ligo/Makefile

```diff
@@ -86,17 +86,25 @@ $(OUT)/trivialDAO_storage.tz: src/**
 	# See "$(OUT)/trivialDAO_storage.tz" for compilation result  #
 	#
 
+# Are we empty? If so produce an empty big_map, otherwise make the TZIP-16 metadata big_map.
+define make-metadata
+$(if $(filter "$(1)$(2)$(3)",""),(Big_map.empty : (string, bytes) big_map),Big_map.literal [("" , Bytes.pack "tezos-storage://$(1)$(if $(filter "$(2)",""),,.$(2))/$(3)")])
+endef
```

Comment:
:thinking: Ligo has `String.concat` and can check if they are empty.
Why not make a function there and use it instead?

We could just pass the 3 arguments to the storage generation functions as-is and deal with them there

Reply:
Done. Did it a bit differently than commented by using the `^` operator instead of `String.concat`, though, and checking with an explicit `if`.